templates/web/base/report/new/category.html

```diff
@@ -28,14 +28,14 @@
     >
         <legend id="form_category_legend">[%~ form_category_label OR loc('Category') ~%]</legend>
         [%~ FOREACH group_or_cat IN category_groups ~%]
-            <div>
+            <div class="govuk-radios__item">
           [% IF group_or_cat.name %]
-                <input required id="category_[% group_or_cat.id %]" type="radio" name="category" value="[% group_or_cat.name %]" data-subcategory="[% group_or_cat.id %]"[% ' checked' IF filter_group == group_or_cat.name %]>
-            <label class="inline" for="category_[% group_or_cat.id %]">[% group_or_cat.name %]</label>
+                <input class="govuk-radios__input" required id="category_[% group_or_cat.id %]" type="radio" name="category" value="[% group_or_cat.name %]" data-subcategory="[% group_or_cat.id %]"[% ' checked' IF filter_group == group_or_cat.name %]>
```

Comment:
If the user attempts to Continue without selecting a radio option, the client-side validation message gets injected before the first radio input, and it looks a bit weird:

![Screenshot_2021-02-04 Reporting a problem](https://user-images.githubusercontent.com/739624/106862308-dd723180-66be-11eb-8a8c-7d95f7f5240e.png)

Do you think we could move it so that it displays _before_ the first `.govuk-radios__item`? (or, same thing but different way of thinking about it, immediately _after_ the `fieldset legend`?)

Reply:
Yeah, the validation code already has a special case for segmented-control which I think I can change to instead check for type==radio which would cover all the segmented-control cases plus this one.